File: CPP_Programming/Day_4/program19.cpp
//Description : Write a program tha accepts number from the user & count its digits

#include<iostream>
using namespace std;

class Digits
{
    public :
        int iNo;

    Digits(int iNo)
    {
        this->iNo = iNo;
    }

    int Count()
    {
        int iDigit = 0;
        int iCount = 0;

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            iCount++;
            iNo = iNo /10;
        }

        return iCount;
    }
};


int main()
{
    int iValue = 0;
    int iRet = 0;

    cout<<"Enter the number : ";
    cin>>iValue;

    Digits dobj(iValue);

    iRet = dobj.Count();

    cout<<"Number of digits are : "<<iRet<<"\n";

    return 0;
}

File: Java_Programming/Day_4/program19.java
//Description : Write a program tha accepts number from the user & count its digits

import java.util.*;
class Digits
{
    public int iNo;

    Digits(int iNo)
    {
        this.iNo = iNo;
    }

    public int Count()
    {
        int iCount = 0;

       while(iNo != 0)
       {
            iCount++;
            iNo = iNo / 10;
       }

       return iCount;
    }

}

class program19
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        Digits dobj = new Digits(iValue);

        int iRet = dobj.Count();

        System.out.println("Number of digits are : "+iRet);

        sobj.close();
    }
}